src/js/popup/popup.js

```diff
@@ -531,6 +530,110 @@ async function enableSettingsPanel() {
   }
 }
 
+async function enableReportIssuePanel() {
+  const reportIssueToggle = document.querySelector(".settings-report-issue");
+  const reportIssueSettingsReturn = document.querySelector(".settings-report-issue-return");
+  const submissionSuccessContinue = document.querySelector(".report-continue");
+  [reportIssueToggle, reportIssueSettingsReturn, submissionSuccessContinue].forEach(e => {
+    e.addEventListener("click", () => {
+      document.body.classList.toggle("show-report-issue");
+      const eventLabel = document.body.classList.contains("show-report-issue") ? "opened-report-issue" : "closed-report-issue";
+      if (document.body.classList.contains("show-report-issue")) {
+        sendRelayEvent("Panel", "click", eventLabel);
+      }
+    });
+  });
+  const reportForm = document.querySelector(".report-issue-content");
+  setURLwithIssue();
+  showReportInputOtherTextField();
+  showSuccessReportSubmission();
+  reportForm.addEventListener('submit', handleReportIssueFormSubmission);
+}
+
+async function handleReportIssueFormSubmission(event) {
+  event.preventDefault();
+  const data = new FormData(event.target);
+  const reportData = Object.fromEntries(data.entries());
+
+  Object.keys(reportData).forEach(function(value) {
+    // Switch "on" to true
+    if (reportData[value] === "on") {
+      reportData[value] = true;
+    }
+    // Remove from report if empty string
+    if (reportData[value] === "") {
+      delete reportData[value];
+    }
+  });
+  // Clean URL data to add "http://" before it if the url doesn't contain a protocol
+  if (!(reportData.issue_on_domain.startsWith("http://") || reportData.issue_on_domain.startsWith("https://"))) {
+    reportData.issue_on_domain = "http://" + reportData.issue_on_domain;
+  }
```

Comment:
This allows the form to accept non-formatted URLs such as `www.test.com` or `test.com`.